test/tools/libtesteth/Options.cpp

```diff
@@ -72,8 +73,29 @@ void printVersion()
 	cout << prepareVersionString() << "\n";
 }
 
-Options::Options(int argc, char** argv)
+Options::Options(int argc, const char** argv)
 {
+    {
+        namespace po = boost::program_options;
+
+        // For some reason boost is confused by -- separator. This extra parser "skips" the --.
+        auto skipDoubleDash = [](const std::string& s) -> std::pair<std::string, std::string> {
+            if (s == "--")
+                return {"--", {}};
+            return {};
+        };
+
+        auto vmOpts = vmProgramOptions();
+        po::parsed_options parsed = po::command_line_parser(argc, argv)
```

Comment:
how do I get back parsed vm option from Options::get() ?

Reply:
You don't. This is only to handle the VM options in single place (VMFactory.cpp). I don't care about the rest of the options. There was a PR introducing program_options in testeth, why is it not merged yet?

Reply:
I believe it was an issue